### DISCUSSION TITLE
launch App but not in app mode as it's buggy - similar to how guacamole does it with a fullscreen black window only displaying the launched apps

### DISCUSSION THREAD

**Discussion by ne0YT** [FreeRDP/FreeRDP#9292] [Q&A]
launch App but not in app mode as it's buggy - similar to how guacamole does it with a fullscreen black window only displaying the launched apps?

is this somehow possible?

**Comment by akallabeth** [comment]
yes, write your own client.
as for the clients currently available directly from `FreeRDP` no.

**Comment by akallabeth** [comment]
further details in https://github.com/FreeRDP/FreeRDP/issues/9304
